main.tf

```diff
@@ -7,7 +9,7 @@ data "aws_iam_policy_document" "assume_role" {
 
     principals {
       type        = "AWS"
-      identifiers = ["arn:aws:iam::${var.master_account_id}:root"]
+      identifiers = ["arn:aws:iam::${var.master_account_id}:root", "arn:aws:iam::${data.aws_caller_identity.current.account_id}:cicd-user"]
```

Comment:
Sorry I didn't ask this earlier, but where is this user created? This does create a dependency on that module, which is OK depend on what else is in there.

Reply:
https://github.com/BetterWorks/terraform-root-modules/pull/14 yeah, i feel like there might be a better solution to handling this, the current solution doesnt feel great but im still building a sense of what a good tf module design looks like